src/components/SearchInput/index.ts

```diff
@@ -0,0 +1 @@
+export { default } from './SearchInput.jsx';
```

Comment:
Our convention in this repo is to just to use index.jsx (or in this case index.tsx) to house the code - can we move the code from SearchInput.jsx into here and then remove that file?